packages/postcss-reduce-idents/src/lib/grid-template.js

```diff
@@ -66,7 +74,7 @@ module.exports = function () {
       declCache.forEach((decl) => {
         decl.value = valueParser(decl.value)
           .walk((node) => {
-            if (/(grid-template|grid-template-areas)/i.test(decl.prop)) {
```

Comment:
`grid-template` already matches `grid-template-areas` but also matches more than intended (namely `grid-template-rows` and `grid-template-columns`).